pkgs/development/python-modules/astropy/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, stdenv 
+, python3Packages
+, buildPythonPackage
+, numpy }:
+
+buildPythonPackage rec {
+    pname = "astropy";
+    version = "1.3.3";
+
+    name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ed093e033fcbee5a3ec122420c3376f8a80f74663214560727d3defe82170a99";
+  };
+  doCheck = false;
```

Comment:
As maintainer its your choice whether or not to enable tests. In any, just leave a comment when you disable them.